### ISSUE TITLE
Opens wrong path (prefixed with home directory)

### ISSUE THREAD

**Issue by taat** [sanduhrs/phpstorm-url-handler#8]
Looks like the .desktop file opens relative path, instead of the absolute one.

When opening:

````
phpstorm://open?file=%2Fhome%2Fuser%2Fwww%2Fapigil%2Fzf%2Fmodule%2FApplication%2Fsrc%2FController%2FIndexController.php&line=38
````

In PhpStorm i get the following error:

File not found:

````
/home/user/./%2Fhome%2Fuser%2Fwww%2Fapigil%2Fzf%2Fmodule%2FApplication%2Fsrc%2FController%2FIndexController.php&line=38
````
(note the `/home/user/./` prefix added)

Is there any way I can fix this?

Latest Ubuntu,
Latest PhpStorm

**Comment by dereuromark** [comment]
I have the same issue in latest phpstorm and linux mint.

Manually calling the CLI `pstorm --line 111 /file/path` works, but not together with the browser click.

**Comment by codingjungle** [comment]
`urldecode() { : "${*//+/ }"; echo -e "${_//%/\\x}"; }
arg=$(urldecode "${arg}")`

adding this after the declaration of arg=${1}, fixes the clicking from the browser (like if you use whoops in your project). full file below, just drop the .txt from it (as github wouldn't let me attach it otherwise).

[phpstorm-url-handler.txt](https://github.com/sanduhrs/phpstorm-url-handler/files/1691467/phpstorm-url-handler.txt)

**Comment by tomasnorre** [comment]
I have the same issue on ubuntu 20.04, the patched version from @codingjungle doesn't work for me.

**Comment by sanduhrs** [comment]
This should be fixed in e46793a6a4270344329d5e5ccbcbc1155f7fa22a
Thanks for reporting.
